### ISSUE TITLE
Publish recommended accessibility checks for contributors

### ISSUE THREAD

**Issue by spacedino** [ampproject/amphtml#17244]
We have some checks that we use internally. I think we should formalize this and post it to github as part of the contributor ecosystem.

I'm not sure if we already do this with good first issues?

cc/ @aghassemi @cramforce @mrjoro @rudygalfi @ericlindley-g

**Comment by choumx** [comment]
/to @CrystalFaith @mrjoro 

Ali should be able to help with a11y for UI components.

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @CrystalFaith Do you have any updates?

**Comment by aghassemi** [comment]
@nainar maybe snippets from the upcoming blog post can be used to craft this?

**Comment by caroqliu** [comment]
Fixed by #30018
